### DISCUSSION TITLE
Lab 1 Software task

### DISCUSSION THREAD

**Discussion by James9306** [NUS-CG3207/labs#5] [Lab 1] [answered]
Hi, my current data segment tab shows that im looking at 0x10010000 (.data), but for the lab the address is at 0x00002000 (.data), so does tha mean we have to manually press the arrow to scroll to that address? Am I doing something wrong? Because it will take quite some time..

**Comment by nicholas132000** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/3b4e8286-e9a6-4301-b3e6-afb973c0c499)

**Reply by James9306** [reply]
thanks bro!
